app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -142,7 +142,15 @@ authselect --useshadow --passalgo=<%= @host.operatingsystem.password_hash.downca
 <% else -%>
 authconfig --useshadow --passalgo=<%= @host.operatingsystem.password_hash.downcase || 'sha256' %> --kickstart
 <% end -%>
+
```

Comment:
I'd have a slight preference to keep this together with the other options. We already have too many nonsense empty lines in the rendered template.
```suggestion
```